### ISSUE TITLE
Restore compile and run functionality

### ISSUE THREAD

**Issue by hinerm** [imagej/imagej-ui-swing#48]
@ctrueden and I have been discussing this in the context of the current [deployment goals](http://imagej.net/2015-06-15_-_Major_updates_in_the_works).

With the changes to [file structure](https://github.com/imagej/imagej/issues/101) and second look at making [install work](https://github.com/imagej/imagej-ui-swing/issues/44) in the script editor, I think it would be nice to get a minimal version of `Compile and Run` working again.

My proposal is to use a dedicated class loader that is re-created during each execution, and only support one-off plugins.

**Comment by ctrueden** [comment]
Yeah, I think it should just call the same functionality that the Script Editor uses when running Java code.
